assets/app/components/site/SideNav/sideNav.js

```diff
@@ -16,28 +16,30 @@ class SideNav extends React.Component {
     const { siteId } = this.props;
 
     return (
-      <ul className="site-actions">
-        <SideNavItem
-          href={this.getUrl(siteId)}
-          icon='pages'
-          linkText={sideNavPaths.PAGES}
-        />
-        <SideNavItem
-          href={this.getUrl(siteId, sideNavPaths.MEDIA)}
-          icon='media'
-          linkText={sideNavPaths.MEDIA}
-        />
-        <SideNavItem
-          href={this.getUrl(siteId, sideNavPaths.SETTINGS)}
-          icon='settings'
-          linkText={sideNavPaths.SETTINGS}
-        />
-        <SideNavItem
-          href={this.getUrl(siteId, sideNavPaths.LOGS)}
-          icon='logs'
-          linkText={sideNavPaths.LOGS}
-        />
-      </ul>
+      <div className="usa-width-one-sixth" id="fool">
```

Comment:
haha I have no idea, I haven't changed any of the original ids. It was helpful when I was searching the code...